identity-service/src/notifications/notificationProcessing.js

```diff
@@ -75,7 +66,8 @@ async function _processSubscriberPushNotifications (tx) {
         entry.notificationTarget,
         tx,
         true,
-        entry.title
+        entry.title,
+        ['mobile', 'browser']
```

Comment:
shouldn't this check if should notify instead of hardcoding both?

Reply:
The subscriber push notification is not part of the settings - so you can't turn it off. 
In this case it will check if the subscriber to the user who posted content has a mobile device registered/browser push notification registered and try to send it. essentially this setting is always true. @hareeshnagaraj can you verify that my logic for this is correct.

Reply:
so you can still disable push notifications even if you are a subscriber right? but this disables _all_ push notifications, not just push notifications from users you have subscribed to.